staff/sergiomons/cv/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Sergio Monge Sosa</h1>
+    <p>Full-Stack Developer(Bueno lo intentaré ser)</p>
+    <h2>Experiencia profesional</h2>
+    <ul>
+        <li><h3>Ingeniería Fran López</h3></li>
```

Comment:
I think it is recommended, not to put headers inside a list tag (Change the font-size of the list tags later on the CSS)

Reply:
I agree with you!! i,m gonna correct it!